core/src/wallet/cosmos/explorers/GaiaCosmosLikeBlockchainExplorer.cpp

```diff
@@ -427,5 +427,42 @@ namespace ledger {
                     });
         }
 
+            // Validators
+        Future<cosmos::ValidatorList> GaiaCosmosLikeBlockchainExplorer::getActiveValidatorSet() const {
+            const bool parseJsonNumbersAsStrings = true;
+            auto basicValidatorList =_http->GET("/staking/validators?status=bonded&page=1&limit=130")
+                .json(parseJsonNumbersAsStrings)
+                .map<cosmos::ValidatorList>(
+                    getContext(), [=](const HttpRequest::JsonResult& response) {
```

Comment:
Any reason for this capture by copy? ([=])

Reply:
Probably not, I'll remove it, especially since the payload is probably big-ish

Reply:
Actually there's no capture at all :D

Reply:
Yes nothing is actually captured ^^ if that's what you mean
The '=' is not necessary, but pretty harmless indeed.